source/_drafts/Introducing-Video-js-HTTP-Streaming-VHS.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introducing Video.js HTTP Streaming (VHS)
+author:
+  name: Garrett Singer
+  github: gesinger
+tags:
+  - vhs
```

Comment:
should the tag be the full name? Maybe `playback` too

Reply:
can be all of the above